examples/css/index.html

```diff
@@ -83,6 +83,88 @@ <h3 class="mistica-text-title2">Tag</h3>
+                <h4 class="mistica-text-title1">Column headings</h4>
+                <table class="mistica-table mistica-table-mobile-collapsed">
+                    <thead>
+                        <tr>
+                            <th>Type of food</th>
+                            <th>Calories</th>
+                            <th>Tasty factor</th>
+                            <th>Price</th>
+                            <th>Average rating</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td data-heading="Type of food">Slice of pizza</td>
+                            <td data-heading="Calories">285</td>
+                            <td data-heading="Tasty factor">95%</td>
+                            <td data-heading="Price">5,00€</td>
+                            <td data-heading="Average rating">9/10</td>
+                        </tr>
+                        <tr>
+                            <td data-heading="Type of food">Hamburger</td>
+                            <td data-heading="Calories">354</td>
+                            <td data-heading="Tasty factor">90%</td>
+                            <td data-heading="Price">6,00€</td>
+                            <td data-heading="Average rating">8/10</td>
+                        </tr>
+                        <tr>
+                            <td data-heading="Type of food">Sushi</td>
+                            <td data-heading="Calories">200</td>
+                            <td data-heading="Tasty factor">85%</td>
+                            <td data-heading="Price">10,00€</td>
+                            <td data-heading="Average rating">10/10</td>
+                        </tr>
+                        <tr>
+                            <td data-heading="Type of food">Salad</td>
+                            <td data-heading="Calories">150</td>
+                            <td data-heading="Tasty factor">70%</td>
+                            <td data-heading="Price">4,00€</td>
+                            <td data-heading="Average rating">7/10</td>
+                        </tr>
+                        <tr>
+                            <td data-heading="Type of food">Ice cream</td>
+                            <td data-heading="Calories">250</td>
+                            <td data-heading="Tasty factor">95%</td>
+                            <td data-heading="Price">3,00€</td>
+                            <td data-heading="Average rating">9/10</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <h4 class="mistica-text-title1">Row headings</h4>
+                <table class="mistica-table mistica-table-mobile-collapsed">
+                    <tbody>
+                        <tr>
+                            <th scope="row"><span class="mistica-text-3">Conectividad</span></th>
```

Comment:
In this example I show a table without thead but with rows heading (th with row scope). In AEM we should allow to configure this.

Also, note how I'm overriding the default font size of cells by nesting a span with the mistica-text-3 class. This kind of customization should be possible in AEM too. Ideally we should allow including some other components inside table cells (like Tags)